tests/extension.test.ts

```diff
@@ -92,5 +92,11 @@ describe('Extension', () => {
       expect((vscode.commands.registerCommand as jest.Mock<any>).mock.calls).toContainEqual(expected)
       expect(context.subscriptions.push.mock.calls[0]).toContain(expected)
     })
+
+    it('should register a DebugConfigurationProvider', () => {
+      activate(context)
+
+      expect(vscode.debug.registerDebugConfigurationProvider).toBeCalled()
```

Comment:
I don't think that we would gain much. TypeScript already enforces that we actually register a `DebugConfigurationProvider`, so in my opinion further specifying the test would be more a sort of burden when changing the behavior than prevent inadvertent changes.

Reply:
this is to protect others from making unintended changes, such as removed registration for "vscode-jest-tests". Since we don't have any other tests for launch-snippets function, I think these simple tests are the least we can do to safeguard its intended logic. 

All tests are creating friction for future changes, and that is not necessary a bad thing...